lib/appmap/class_map.rb

```diff
@@ -141,6 +142,22 @@ def add_function(root, method, include_source: true)
         end
       end
 
+      # Labels can be embedded in the function comment. Label format is similar to YARD and JavaDoc.
+      # The keyword is @labels or @label. The keyword is followed by space-separated labels.
+      # For example:
+      # @label provider.authentication security
+      def parse_labels(comment)
+        return [] unless comment
+
+        comment
+          .split("\n")
+          .map { |line| line.match(/^\s*#\s*@labels?\s+(.*)/) }
```

Comment:
Is there any value to using something like https://ruby.github.io/rdoc/RDoc/Text.html#method-i-normalize_comment to ensure that the comment is going to be formatted in a way we'll understand?

Reply:
Hmm, I guess there may be some comment forms that I don’t know about? I’m just covering the common case 

```ruby
  # @labels foo bar
```

Reply:
Probably fine for now. We may want to revisit if we find that there are other forms we need to handle -- it might make sense to leverage some other work, rather than making this code more complicated.